### PR TITLE
Update mobs sea creatures

### PR DESCRIPTION
**Pull request by catgirlseraid**
umm i did kind of mess up the commits sorry so i hope this turns out ok 

went through the rest of the sea creatures and updated their drops

### CODE REVIEW

File: items/FROZEN_STEVE_SC.json [hunk 1]
```diff
@@ -69,6 +69,10 @@
           "id": "ICE_ROD",
           "chance": "4%"
         },
+        {
+          "id": "ESSENCE_ICE",
+          "chance": "x0-1"
```
Comment (NopoTheGamer) [inline items/FROZEN_STEVE_SC.json:74]: why isnt this a percent

File: items/REINDRAKE_SC.json [hunk 2]
```diff
@@ -35,22 +36,27 @@
...
         {
           "id": "PROSPERITY;1",
           "extra": [
             "Only drops for the player who spawned the Reindrake"
           ],
           "chance": "100%"
+        },
+        {
+          "id": "DYE_ICEBERG",
+          "extra": [],
```
Comment (NopoTheGamer) [inline items/REINDRAKE_SC.json:58]: empty extra?

Comment (catgirlseraid) [reply items/REINDRAKE_SC.json:58]: {85A5B0CB-C046-4882-9DBD-98290B7C32F4}

im really not sure why it's done like that either, i don't mind removing it but it's just copied and pasted from another mob

File: items/FROZEN_STEVE_SC.json (hunk 1 above)
Comment (catgirlseraid) [reply items/FROZEN_STEVE_SC.json:74]: {E417CEFB-034F-45B9-8907-A7D1245C75BB}

wiki says 25% though so changed it, thanks

**Pull request merged**